.github/workflows/OperatorIntegrtionTest.yml

```diff
@@ -0,0 +1,84 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+name: OperatorIntegrationTest.yml
```

Comment:
Can we change the name of this workflow to follow the same convention as the others?

`operator-integration-test.yml`